app/src/main/res/layout/all_subject_week.xml

```diff
@@ -0,0 +1,148 @@
+        app:layout_constraintTop_toTopOf="@id/top_guideline"
+        android:gravity="center_horizontal"
+        android:orientation="horizontal">
+
+        <androidx.appcompat.widget.AppCompatButton
+            android:id="@+id/button_name_university"
+            android:layout_width="232dp"
+            android:layout_height="54dp"
+            android:layout_margin="15dp"
+            android:background="@drawable/btn_radius_10dp"
+            android:backgroundTint="@color/major"
+            android:fontFamily="@font/poppins_bold"
+            android:text="@string/week_shedule"
+            android:textAllCaps="false"
+            android:textColor="@color/black"
+            android:textSize="20sp"
+            app:layout_constraintStart_toStartOf="@id/start_guideline"
+            
+            />
+
+        <androidx.appcompat.widget.AppCompatButton
+            android:id="@+id/button_plus_subject"
+            android:layout_margin="15dp"
+            android:layout_width="51dp"
+            android:layout_height="50dp"
+            android:background="@drawable/add_botton"
+            android:backgroundTint="@color/major"
+            app:layout_constraintStart_toStartOf="@id/button_name_university"
+            tools:ignore="SpeakableTextPresentCheck,TouchTargetSizeCheck" />
+    </LinearLayout>
+
+    <ImageView
+        android:id="@+id/shape"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:visibility="visible"
+        android:src="@drawable/shape"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <LinearLayout
+        android:id="@+id/linearLayout3"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/major"
+        android:orientation="vertical"
+        app:layout_constraintBottom_toBottomOf="@id/bottom_guideline"
+        app:layout_constraintEnd_toEndOf="@id/end_guideline"
+        app:layout_constraintStart_toStartOf="@id/start_guideline"
+        app:layout_constraintTop_toBottomOf="@+id/header_btn"
+        app:layout_constraintVertical_bias="0.497">
+
+        <TableRow
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            android:gravity="center">
+
+            <TextView
+                android:id="@+id/subjectDayWeekText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/poppins_bold"
+                android:padding="10dp"
+                android:text="@string/subject_in_a_week"
+                android:textAlignment="center"
+                android:textColor="@color/white"
+                android:textSize="24sp" />
+
+            <androidx.appcompat.widget.AppCompatButton
+                android:id="@+id/btnUp_for_subject"
```

Comment:
Choose the one style for naming, please. btnUpForSubject or btn_up_for_subject.
It also applies to other views.

Reply:
Your are right, i will fix it.